cli/flox-rust-sdk/src/models/environment/managed_environment.rs

```diff
@@ -101,7 +101,7 @@ pub enum ManagedEnvironmentError {
     #[error("access to floxmeta repository was denied")]
     AccessDenied,
     #[error("environment '{0}' does not exist at upstream '{1}'")]
-    UpstreamNotFound(EnvironmentRef, String),
+    UpstreamNotFound(EnvironmentRef, String, Option<String>),
```

Comment:
Suggestion: make this a struct variant with named fields, it's unclear what the `Option<String>` is supposed to represent from looking at this.

Something like this
```rust
#[error("environment '{name}' does not exist at upstream '{upstream}'")]
UpstreamNotFound {
    name: EnvironmentRef,
    upstream: String,
    whatever_this_one_is: Option<String>,
}
```